tests/countries/test_turkmenistan.py

```
@@ -0,0 +1,119 @@
+        self.assertHolidayName("Гурбан байрамы (estimated)", "2024-06-16")
+        self.assertHolidayName("Гурбан байрамы (estimated)", "2025-06-06")
+
+    def test_localization(self):
+        try:
+            turkmen_holidays = Turkmenistan(years=range(1992, 2050), language="tk")
+            for holiday in turkmen_holidays:
+                self.assertEqual(
+                    holiday.lang, "tk", f"Holiday {holiday.name} not localized in Turkmen."
+                )
+        except FileNotFoundError:
+            print("Turkmen language translation files not found")
```

Ruff (0.8.2)

`print` found

Remove `print`

(T201)